src/test/java/org/opentripplanner/middleware/BugsnagTest.java

```diff
@@ -0,0 +1,59 @@
+package org.opentripplanner.middleware;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.middleware.bugsnag.BugsnagReporter;
+import org.opentripplanner.middleware.models.BugsnagEvent;
+import org.opentripplanner.middleware.models.BugsnagEventRequest;
+import org.opentripplanner.middleware.persistence.Persistence;
+import org.opentripplanner.middleware.utils.FileUtils;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class BugsnagTest extends OtpMiddlewareTest {
```

Comment:
An older version of this file seemed to be able to send events to Bugsnag. Where is this happening now?

Reply:
@binh-dam-ibigroup I removed that test because it meant having a BUGSNAG_PROJECT_NOTIFIER_API_KEY in the (now renamed) env.yml.tmp file which will get commited. Looking at the .gitignore file, perhaps this entry should be removed to allow this test?:

# Test config is OK.
!configurations/test/env.yml

Reply:
I was looking for a call to report an event to Bugsnag, and you have addressed that question.

Going forward, it makes sense to have an entry `BUGSNAG_PROJECT_NOTIFIER_API_KEY` in `configurations/test/env.yml.tmp`, that way both the `default` and `test` configuration templates are consistent. (You don't want to check in the actual `.yml` files as discussed in company chat.)